lib/solargraph/source_map.rb

```diff
@@ -141,7 +141,7 @@ def references name
     # @return [Array<Pin::LocalVariable>]
     def locals_at(location)
       return [] if location.filename != filename
-      closure = locate_named_path_pin(location.range.start.line, location.range.start.character)
+      closure = locate_closure_pin(location.range.start.line, location.range.start.character)
```

Comment:
Locals ought to flow from any closure (that's the definition of closure as I understand it - something that can close upon locals), including anonymous blocks